normandy/base/middleware.py

```diff
@@ -146,3 +146,23 @@ def middleware(request):
         return response
 
     return middleware
+
+
+def show_yaml_in_browser_middleware(get_response):
```

Comment:
That would work, but wouldn't quite do the same thing as I have here. I'd prefer to keep the yaml content type for consumers that understand it, and only switch it to `text/plain` for browsers. As far as I can tell there isn't a way to make a renderer with a dynamic content type (which is honestly probably for the best to avoid hooligans like me doing things like dynamic content types).